tgui-next/packages/tgui/interfaces/MiningVendor.js

```diff
@@ -0,0 +1,64 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { Section, Box, Button, Table } from '../components';
+import { classes } from 'common/react';
+
+export const MiningVendor = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  let inventory = [
+    ...data.product_records,
+  ];
+  return (
+    <Fragment>
+      <Section title="User">
+        {data.user && (
+          <Box>
+          Welcome, <b>{data.user.name || "Unknown"}</b>,
+            {' '}
```

Comment:
`&nbsp;`  ?

Reply:
I'd keep using `{' '}`, because spaces should be breakable.

Reply:
`{' '}` is also used in many other tgui-next interfaces, while there are none with `&nbsp;`  in it, so I would also rather feel more comfortable with this solution.

Reply:
Alright, it's not much of a difference anyway.